IFC4x3/PropertyEnumerations/a/PEnum_ArrangerDirectionEnum.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyEnumeration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="PEnum_ArrangerDirectionEnum" Name="PEnum_ArrangerDirectionEnum" UniqueId="cd999af4-25bd-44bd-a741-6dc54e5c0d45">
+	<Constants>
+		<DocPropertyConstant xsi:nil="true" href="FRONT_3mBnqQ9KfDA8NUM3f7H6vi" />
+		<DocPropertyConstant xsi:nil="true" href="HORIZONTAL_1WJWlNNxH41wlZWDytiBt_" />
+		<DocPropertyConstant xsi:nil="true" href="REAR_3qjtx2stH4pBW2sacnr0nx" />
+		<DocPropertyConstant xsi:nil="true" href="VERTICAL_25mDcgj54eEwz8mztCbb" />
```

Comment:
Existing - `VERTICAL_3JFphC50n1DvVmLTJFLQZI`
Documentation: Vertical orientation or direction

Reply:
added

Reply:
again @czapplitec  these seem to not be updated

Reply:
Marking